nomad-clouddriver/src/main/kotlin/io/armory/plugin/nomad/validators/UpsertNomadJobDescriptionValidator.kt

```diff
@@ -8,6 +8,10 @@ import org.springframework.validation.Errors
 
 @NomadOperation(NomadOperations.UpsertJob)
 class UpsertNomadJobDescriptionValidator : DescriptionValidator<UpsertNomadJobDescription>(){
+    /**
+     * More nullable types. This is more the fault of the existing interfaces, but we should `@NullableByDefault` the
+     * `clouddriver-api` package-info so it's clearer for developers that they don't need to null-check.
+     */
```

Comment:
I totally agree about the nullable types. I think that is only done where the current contracts enforce that.